superset-frontend/packages/superset-ui-chart-controls/src/types.ts

```diff
@@ -340,11 +341,26 @@ export interface ControlPanelSectionConfig {
   controlSetRows: ControlSetRow[];
 }
 
+export interface SharedFormData {
```

Comment:
Would it make sense to call this `NormalizedFormData` or similar? "Shared" feels slightly too generic, and "normalized" or "standardized" would more clearly indicate that a transformation has taken place. To disambiguate with the already existing `StandardizedFormDataInterface`, maybe we should consider also renaming that to `StandardizedState`, as it's not directly `FormData`, but a kind-of struct containing "generic stuff" that's needed to perform the normalization operations.

Reply:
Yes, the `sharedformData` is not an appropriate name here. Maybe `SharedControlls` / `StandardizedState` is more direct. I am also considering whether should we rename the `StandardizedFormData` -> `StandardizedFormDataState` or `IntermediateFormData`. How about you think?

Reply:
I like `standardizedState` or `normalizedState` 👍